packages/eds-core-react/src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -195,6 +195,28 @@ const findPrevIndex: IndexFinderType = ({
   return prevIndex
 }
 
+type OptionLabelProps<T> = T extends string | number
```

Comment:
with this solution i am getting this error 
![bilde](https://github.com/equinor/design-system/assets/4499531/d0e2b0f4-cac8-449c-b248-00eaa8e32eb9)

Reply:
Can you try and restart your typescript server after checking out the changes? I am not getting the same error

![image](https://github.com/equinor/design-system/assets/25079611/bd44a849-180b-44e5-b4f6-f1041132ddb0)

Reply:
Ah yes you are quite right! Everything seems to be in order now